drivers/test/build.gradle

```diff
@@ -1,10 +1,28 @@
 // TODO: Multiplatform this so it can be reused when we make non-jvm drivers.
-apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
 
-dependencies {
-  api project(':sqldelight-runtime')
-  
-  api deps.kotlin.stdlib.jdk
-  api deps.junit
-  api deps.truth
+kotlin {
+  sourceSets {
+    commonMain {
+      dependencies {
+        api project(':sqldelight-runtime')
+
+        api deps.kotlin.stdlib.jdk
+        api deps.kotlin.test.common
+        api deps.kotlin.test.commonAnnotations
+      }
+    }
+    jvmMain {
+      dependencies {
+        api deps.kotlin.test.junit
+      }
+    }
+  }
+
+  targets {
+    fromPreset(presets.jvm, 'jvm')
+    fromPreset(presets.js, 'js')
+    fromPreset(presets.iosX64, 'iosX64')
+    fromPreset(presets.iosArm64, 'iosArm64')
```

Comment:
We should figure out a way to keep these lists in sync

Reply:
yea good point. might be able to apply a gradle script in this block?